src/current/_includes/v23.1/misc/assume-role-description.md

```diff
@@ -0,0 +1 @@
+Pass the [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the role to assume. Use in combination with `AUTH=implicit` or `specified`. <span class="version-tag">New in v23.1:</span> `external_id`: Use as a value to `ASSUME_ROLE` to specify the [external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html) for third-party access to your S3 bucket.
```

Comment:
Do you think ASSUME_ROLE should be linked to: https://deploy-preview-18052--cockroachdb-docs.netlify.app/docs/v23.1/cloud-storage-authentication#amazon-s3-assume-role?
Or maybe:
> Refer to [Amazon S3 assume role]({% link {{ page.version.version }}/cloud-storage-authentication.md %}#amazon-s3-assume-role) for setup details.